WordPress/Classes/Services/PostService.m

```diff
@@ -593,6 +389,82 @@ - (AbstractPost *)findPostWithID:(NSNumber *)postID inBlog:(Blog *)blog {
     return [posts firstObject];
 }
 
+- (NSDictionary *)remoteSyncParametersDictionaryForRemote:(nonnull id <PostServiceRemote>)remote
```

Comment:
Rather than account for differences in remote parameters here, it would probably be better to let the remotes handle it themselves. It would avoid dealing with extra dependencies if the remotes change, a new remote is added, or an old one removed.